### ISSUE TITLE
Python3 porting checker do not detect the bytes can only contain ASCII literal characters.

### ISSUE THREAD

**Issue by coldnight** [PyCQA/pylint#1545]
### Steps to reproduce
1. Save the following content to `test.py`:
    ```python
    text = b"测试"
    ```
2.  `python3 test.py` output:
  ```text
   File "test.py", line 6
      text = b"测试"
            ^
  SyntaxError: bytes can only contain ASCII literal characters.
 ```
3. `pylint --py3k test.py` output:

  ```text

  --------------------------------------------------------------------
  Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

   ```

### Current behavior

Do not detect the bytes can only contain ASCII literal characters.

### Expected behavior

 Detects the bytes can only contain ASCII literal characters.

### pylint --version output

```
pylint 1.7.1,
astroid 1.5.2
Python 2.7.13 (default, Dec 18 2016, 07:03:39)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
```

**Comment by PCManticore** [comment]
Thanks, this could be a nice addition to the Python 3 porting checker, but only if we can infer bytes accordingly.
